scripts/apps-version-check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+latest_tag=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+
+bad_app_count=0
+
+while read -r app; do
+    if [ "$app" != "emqx" ]; then
+        app_path="$app"
+    else
+        app_path="."
+    fi
+    src_file="$app_path/src/$(basename "$app").app.src"
+    app_version=$(grep -E 'vsn' "$src_file" | grep -oE '[0-9]+\.[0-9]+(-alpha|-beta|-rc)?\.[0-9]+')
```

Comment:
we do not use alpha beta and rc in app vsns
```suggestion
    app_version=$(grep -E 'vsn' "$src_file" | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
```